iocore/net/quic/QUICCrypto.h

```diff
@@ -0,0 +1,121 @@
+/** @file
```

Comment:
One thing we're excited about is TLS 1.3 handshake. It should be available not only with QUIC but also HTTP.